docs/ref/databases.txt

```diff
@@ -941,6 +941,27 @@ To enable the JSON1 extension you can follow the instruction on
 .. _JSON1 extension: https://www.sqlite.org/json1.html
 .. _the wiki page: https://code.djangoproject.com/wiki/JSON1Extension
 
+.. _sqlite-init-command:
+
+Setting PRAGMA Options
```

Comment:
```suggestion
Setting PRAGMA options
```

Reply:
As far as I'm aware, `pragma` should be uppercased only when we talk about an SQL statement.
```suggestion
Setting pragma options
```

Reply:
Corrected